packages/kit/test/build-errors/package.json

```diff
@@ -0,0 +1,12 @@
+{
+	"name": "test-meta",
```

Comment:
Should rename this to `build-errors` (will do after current test suite finishes)